bin/foreman_scap_client

```diff
@@ -46,22 +46,66 @@ ensure_scan_files () {
 }
 
 load_config () {
-  local config_file="/etc/foreman_scap_client/config"
+  local config_file="/etc/foreman_scap_client/config.yaml"
 
-  # Load config file
   if [ -f $config_file ]; then
-    # shellcheck disable=SC1090
-    source $config_file
+    eval "$(awk '
+    /^:server:/ {
+      print "SERVER=" $2;
+    }
+    /^:port:/ {
+      print "PORT=" $2;
+    }
+    /^:timeout:/ {
+      print "TIMEOUT=" $2;
+    }
+    /^:fetch_remote_resources:/ {
+      print "FETCH_REMOTE_RESOURCES=" $2;
```

Comment:
Do we expect a specific value here? Yaml is somewhat loose about how booleans can be represented. What would happen if the config file had the following bit in it?
```yaml
:fetch_remote_resources: no
```